build-tools/Xamarin.Android.Tools.BootstrapTasks/result-packaging.targets

```diff
@@ -24,13 +24,10 @@
   <ItemGroup>
     <_TestResultFiles Include="$(XamarinAndroidSourcePath)TestResult-*.xml" />
     <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\TestResult-*.xml" />
-    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\*.log" />
     <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\compatibility\*" />
     <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\logcat*" />
-    <_TestResultFiles Include="$(XamarinAndroidSourcePath)bin\Test$(Configuration)\msbuild*.binlog*" />
```

Comment:
Is the trailing `*` in `.binlog*` important? I didn't know why it was here.

Reply:
It was pulled from https://github.com/xamarin/xamarin-android/blob/decfbccf3e99449c9a8d6afef113c9339f1bc85b/build-tools/scripts/Packaging.mk#L79, but I think it can be removed.

That being said, I have some related changes staged in https://github.com/xamarin/xamarin-android/pull/3042/files#diff-d52b2bac3b90f1611ffdb03eb6112d70R29. Perhaps we can just apply these changes there? It should be ready to land ~today.

Reply:
We can just rebase depending on which is merged first. Your PR is doing lots of other stuff, probably fine as it is.